apps/project-renderer/src/components/app/style.scss

```diff
@@ -0,0 +1,13 @@
+@import '~@wordpress/block-library/build-style/style.css';
```

Comment:
Nice! Yeah, this looks like it should solve our problems 👍  🙇 

That said, maybe worth leaving a note regarding the future: eventually, we would like to be able to serve Gutenberg separately from our app - similar to how WP does it but it's currently harder for us to do without a WP backend.  
But when we get there, I assume we'd just host a `@wordpress/block-library` `*.css` target somewhere and pull that as a regular stylesheet. Until then, this should serve us well.

Reply:
Great! Should we add this note somewhere else?

Reply:
Hmm... I added a reference to this conversation in c/3mQAfwZg-tr.